financial-services-accelerator/components/org.wso2.financial.services.accelerator.consent.mgt.extensions/src/main/java/org/wso2/financial/services/accelerator/consent/mgt/extensions/manage/model/ExternalAPIConsentGenerateResponseDTO.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ * <p>
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.financial.services.accelerator.consent.mgt.extensions.manage.model;
+
+import java.util.Map;
+
+/**
+ * ExternalAPIConsentGenerateResponseDTO
+ */
+public class ExternalAPIConsentGenerateResponseDTO {
+
+    private Map<String, Object> receipt;
+    private int consentFrequency;
+    private long validityTime;
+    private boolean recurringIndicator;
+    private String consentStatus;
+    private String consentType;
+    private String authorizationStatus;
+    private String authorizationType;
```

Comment:
in consent generation,is there a need to change authorizationType and authorizationStatus

Reply:
These status values are the ones stored in database. We can give the extension developer a choice to modify them. Maybe we can add default values and make these optional.